external/fv3fit/fv3fit/pytorch/predict.py

```diff
@@ -343,9 +343,12 @@ def _pack_to_tensor(
                 n_windows, timesteps, *data.shape[1:]
             )
             # append first time of next window to end of each window
-            end_data = np.concatenate(
-                [data[1:, :1, :], normalized_data[None, -1:, :]], axis=0
-            )
+            if n_windows > 1:
+                end_data = np.concatenate(
+                    [data[1:, :1, :], normalized_data[None, -1:, :]], axis=0
```

Comment:
Before the first entry data[1:] would be of size zero, and `np.concatenate` complained.

Reply:
Is the functionality here basically the same as in the `WindowedZarrLoader`? Could the code not be repeated?